### PR TITLE
HW #4 - Functions

### PR DESCRIPTION
**Pull request by omgjulie**

### CODE REVIEW

File: js_functions/task1.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// function declaration
+
+function countRectArea1 (width, height) {
```
Comment (sttaran) [inline js_functions/task1.js:3]: calculateArea would be better name IMO

File: js_functions/task2.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+const checkPersonAge = age => age >= 18;
```
Comment (sttaran) [inline js_functions/task2.js:1]: We should name it somehow so I can understand that the function returns boolean value, for example `checkIfAdult`, `getIsAdult` etc

File: js_functions/task3.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const makeRecursion = num => {
+    console.log(num);
+    if (num <= 0) {
```
Comment (sttaran) [inline js_functions/task3.js:3]: function should not be called when num is 0

Фунція повинна викликатися, доки num не стане менше або рівне 0.

**Review by sttaran — CHANGES_REQUESTED** [review]

File: js_functions/task3.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const makeRecursion = num => {
```
Comment (sttaran) [inline js_functions/task3.js:1]: function name should represent what the function does. e.g. `countDown`

**Review by sttaran — CHANGES_REQUESTED** [review]

File: js_functions/task2.js (hunk 2 above)
Comment (omgjulie) [reply js_functions/task2.js]: changed the name of function

File: js_functions/task1.js (hunk 1 above)
Comment (omgjulie) [reply js_functions/task1.js]: changed the name of function

File: js_functions/task3.js (hunk 3 above)
Comment (omgjulie) [reply js_functions/task3.js]: changed the condition 
`if (num < 0)`

File: js_functions/task3.js (hunk 4 above)
Comment (omgjulie) [reply js_functions/task3.js]: changed the name of function

**Pull request merged**